sdk/communication/Azure.ResourceManager.Communication/src/Generated/SuppressionListResourceData.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.Core;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.Communication
+{
+    /// <summary>
+    /// A class representing the SuppressionListResource data model.
+    /// A class representing a SuppressionList resource.
+    /// </summary>
+    public partial class SuppressionListResourceData : ResourceData
+    {
+        /// <summary> Initializes a new instance of SuppressionListResourceData. </summary>
+        public SuppressionListResourceData()
+        {
+        }
+
+        /// <summary> Initializes a new instance of SuppressionListResourceData. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="listName"> The the name of the suppression list. This value must match one of the valid sender usernames of the sending domain. </param>
+        /// <param name="lastUpdatedTimeStamp"> The date the resource was last updated. </param>
+        /// <param name="createdTimeStamp"> The date the resource was created. </param>
+        /// <param name="dataLocation"> The location where the SuppressionListAddress data is stored at rest. This value is inherited from the parent Domains resource. </param>
+        internal SuppressionListResourceData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string listName, string lastUpdatedTimeStamp, string createdTimeStamp, string dataLocation) : base(id, name, resourceType, systemData)
+        {
+            ListName = listName;
+            LastUpdatedTimeStamp = lastUpdatedTimeStamp;
+            CreatedTimeStamp = createdTimeStamp;
+            DataLocation = dataLocation;
+        }
+
+        /// <summary> The the name of the suppression list. This value must match one of the valid sender usernames of the sending domain. </summary>
+        public string ListName { get; set; }
+        /// <summary> The date the resource was last updated. </summary>
+        public string LastUpdatedTimeStamp { get; }
```

Comment:
Could this be a type of `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).

Reply:
same as `CreatedTimeStamp` below

Reply:
Please take a look at the changes.  Autorest did not detect these changes

  SuppressionListProperties.createdTimeStamp: -|date-time
  SuppressionListProperties.lastUpdatedTimeStamp: -|date-time

Reply:
> Please take a look at the changes. Autorest did not detect these changes
> 
> SuppressionListProperties.createdTimeStamp: -|date-time SuppressionListProperties.lastUpdatedTimeStamp: -|date-time

@matthohn-msft It should be:

```
SuppressionListResource.properties.createdTimeStamp: -|date-time
SuppressionListResource.properties.lastUpdatedTimeStamp: -|date-time
```

I've commented in the changes in `autorest.md`.